src/libraries/System.DirectoryServices.Protocols/src/System.DirectoryServices.Protocols.csproj

```diff
@@ -70,7 +70,7 @@
       <Link>Common\Interop\Linux\OpenLdap\Interop.Ber.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsLinux)' == 'true'">
+  <ItemGroup Condition="'$(TargetsLinux)' == 'true' or '$(TargetsBrowser)' == 'true'">
```

Comment:
I believe we said this should be PNSE (as rid-less impl).  https://github.com/dotnet/runtime/pull/37944#discussion_r442409000